prometheus-to-sd/Dockerfile

```diff
@@ -12,10 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM gcr.io/google-containers/debian-base-amd64:0.3
+FROM golang:1.9-alpine as builder
+ARG DEP_VERSION="0.4.1"
+RUN apk update && apk add bash git
+ADD https://github.com/golang/dep/releases/download/v${DEP_VERSION}/dep-linux-amd64 /usr/bin/dep
+RUN chmod +x /usr/bin/dep
+WORKDIR ${GOPATH}/src/github.com/GoogleCloudPlatform/k8s-stackdriver/prometheus-to-sd
+COPY Gopkg.toml Gopkg.lock ./
+RUN dep ensure -vendor-only
```

Comment:
I don't like idea of using dep when building image. Can we just use preexisting vendor?

Reply:
it is used only to verify dependencies, I think it is good to have one more step of check

Reply:
verifying dependencies could be part of test pipeline not building image